### PR TITLE
Feature/clogs config

### PR DESCRIPTION
**Pull request by parc-jason**
# Description
Some additional changes to the common-logging.
Added more configuration values (change layout and naming of environment variables).
Allow the caller to specify a 'env', which we can leverage for indices or filters.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-idcqvl-tools.pathfinder.gov.bc.ca/dashboard?id=common-document-generation-service-pr-26

File: app/src/common-logging/common-logging-transformer.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    this._metadata = options && options.metadata && options.metadata === Object(options.metadata) ? options.metadata : undefined;
+    this._data = options && options.data && options.data === Object(options.data) ? options.data : undefined;
+  }
+
+  xform(message, options) {
+    if (!message) {
+      return;
+    }
+    let opts = options || {};
```
Comment (jujaga) [inline app/src/common-logging/common-logging-transformer.js]: Doesn't look like opts is reassigned - probably could be declared as a const instead

File: app/src/common-logging/common-logging-constants.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (parc-jason) [reply app/src/common-logging/common-logging-constants.js]: Good catch!

File: app/src/common-logging/common-logging-transformer.js (hunk 1 above)
Comment (parc-jason) [reply app/src/common-logging/common-logging-transformer.js]: got it

**Pull request merged**